modules/microvm/virtualization/microvm/microvm-host.nix

```diff
@@ -90,23 +100,58 @@ in
           };
 
     })
-    (lib.mkIf cfg.sharedVmDirectory.enable {
+    (mkIf cfg.sharedVmDirectory.enable {
       ghaf.virtualization.microvm.guivm.extraModules = [ (import ./common/shared-directory.nix "") ];
 
       # Create directories required for sharing files with correct permissions.
       systemd.tmpfiles.rules =
         let
           vmDirs = map (
             n:
-            "d /storagevm/shared/shares/Unsafe\\x20${n}\\x20share/ 0700 ${config.ghaf.users.accounts.user} users"
+            "d /storagevm/shared/shares/Unsafe\\x20${n}\\x20share/ 0760 ${toString config.ghaf.users.loginUser.uid} users"
           ) cfg.sharedVmDirectory.vms;
         in
         [
           "d /storagevm/shared 0755 root root"
-          "d /storagevm/shared/shares 0700 ${config.ghaf.users.accounts.user} users"
+          "d /storagevm/shared/shares 0760 ${toString config.ghaf.users.loginUser.uid} users"
         ]
         ++ vmDirs;
-
     })
+    (mkIf config.ghaf.profiles.debug.enable {
+      # Host service to remove user
+      systemd.services.remove-users =
+        let
+          userRemovalScript = pkgs.writeShellApplication {
+            name = "remove-users";
+            runtimeInputs = [
+              pkgs.coreutils
+            ];
+            text = ''
+              echo "Removing ghaf login user data"
+              rm -r /storagevm/homes/*
+              rm -r /storagevm/gui-vm/var/
+              echo "All ghaf login user data removed"
+            '';
+          };
+        in
+        mkIf config.ghaf.profiles.debug.enable {
+          description = "Remove ghaf login users";
+          enable = true;
+          path = [ userRemovalScript ];
+          unitConfig.ConditionPathExists = "/storagevm/gui-vm/var/lib/nixos/user.lock";
+          serviceConfig = {
```

Comment:
**StandardOutput** defaults to the value of **DefaultStandardOutput**, which itself defaults to **journal**.
Therefore, we can probably remove `StandardOutput = "journal"` and `StandardError = "journal"`, unless they are needed for clarity.

Ref.
https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html